src/include/session.h

```diff
@@ -94,7 +95,8 @@ struct __wt_session_impl {
     u_int ncursors;                  /* Count of active file cursors. */
     uint32_t cursor_sweep_position;  /* Position in cursor_cache for sweep */
     uint32_t cursor_sweep_countdown; /* Countdown to cursor sweep */
-    uint64_t last_cursor_sweep;      /* Last sweep for dead cursors */
+    uint64_t last_cursor_sweep;      /* Last regular sweep for dead cursors */
+    uint64_t last_cursor_big_sweep;  /* Last big sweep for dead cursors */
```

Comment:
Swap for alphabetizing.

Reply:
Should we take the opportunity to swap the `cursor_sweep` ones as well?